rllib/utils/replay_buffers/replay_buffer.py

```diff
@@ -221,10 +221,11 @@ def add(self, batch: SampleBatchType, **kwargs) -> None:
         elif self.storage_unit == StorageUnit.EPISODES:
             for eps in batch.split_by_episode():
                 if (
-                    eps.get(SampleBatch.T)[0] == 0
-                    and eps.get(SampleBatch.DONES)[-1] == True  # noqa E712
+                    eps.get(SampleBatch.T, [0])[0] == 0
```

Comment:
This is seemingly unrelated but dreamer seems to be the only algo for now that produces an error here.

Reply:
Hmm, maybe view requirements related. But yeah, at least it's fixed for now.